src/rx5808-pro-diversity/channels.cpp

```diff
@@ -0,0 +1,246 @@
+#include <avr/pgmspace.h>
+
+#include "channels.h"
+#include "settings.h"
+
+
+// Channels to sent to the SPI registers
+static const uint16_t channelTable[] PROGMEM = {
+    #define _CHANNEL_REG_FLO(f) ((f - 479) / 2)
+    #define _CHANNEL_REG_N(f) (_CHANNEL_REG_FLO(f) / 32)
+    #define _CHANNEL_REG_A(f) (_CHANNEL_REG_FLO(f) % 32)
+    #define CHANNEL_REG(f) (_CHANNEL_REG_N(f) << 7) | _CHANNEL_REG_A(f)
```

Comment:
Yeah! I was hoping to add an extra mode to the search for fine tuning --- replacing it all with just the values would make sense.